### ISSUE TITLE
Great project. Will the SparseInst algorithm continue to be updated, it seems incomplete?

### ISSUE THREAD

**Issue by Byronnar** [Spritaro/condinst_tensorrt#2]
Great project. Will the SparseInst algorithm continue to be updated, it seems incomplete?

**Comment by Spritaro** [comment]
Unfortunately, the model turned out to be quite difficult to train. It required large batch size (>64), didn't really work well with gradient accumulation, was unstable with half precision... I'd recommend to use the official implementation.

**Comment by Byronnar** [comment]
Thanks, I will try.

**Comment by Byronnar** [comment]
> Unfortunately, the model turned out to be quite difficult to train. It required large batch size (>64), didn't really work well with gradient accumulation, was unstable with half precision... I'd recommend to use the official implementation.

I also want to ask whether the reproduction of CondInst can achieve the accuracy of the paper? I see that this open source warehouse only wrote about 20 mAPs. Is it because the training time is not enough?

**Comment by Spritaro** [comment]
Tweaking training parameters like batch size or data augmentation might increase accuracy. But this implementaiton of CondInst is probably too simple to reproduce the official results. For the sake of simplicity, I used CenterNet without deformable conv layers to detect instances, instead of FCOS as in the CondInst paper.

**Comment by Byronnar** [comment]
OK，Thank you.
